### PR TITLE
Add a tool to auto-ni assignees or triage_owner when a bug has leave-…

### PR DESCRIPTION
**Pull request by calixteman**
…open keyword and no activity for 6 months

### CODE REVIEW

File: auto_nag/scripts/leave_open_no_activity.py [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+from auto_nag.bzcleaner import BzCleaner
+from auto_nag.people import People
+from auto_nag import utils
+
+
+class LeaveOpenNoActivity(BzCleaner):
+    def __init__(self):
+        super(LeaveOpenNoActivity, self).__init__()
+        self.people = People()
+        self.nmonths = utils.get_config(self.name(), 'months_lookup', 6)
```
Comment (sylvestre) [inline auto_nag/scripts/leave_open_no_activity.py]: Can you remove the 6 as we have that in the config file?
I found that confusing.

Comment (calixteman) [reply auto_nag/scripts/leave_open_no_activity.py]: It's just the default value

File: templates/ni_for_leave_open_no_activity_comment.txt [hunk 2]
```diff
@@ -0,0 +1,2 @@
+The leave-open keyword is there and there is no activity for {{ extra['nmonths'] }} {{ plural('month', extra['nmonths']) }}.
+:{{ nickname }}, maybe it's time to close this bug ?
```
Comment (sylvestre) [inline templates/ni_for_leave_open_no_activity_comment.txt]: bug ? => bug?

File: auto_nag/scripts/leave_open_no_activity.py (hunk 1 above)
Comment (sylvestre) [reply auto_nag/scripts/leave_open_no_activity.py]: I know but what is the point as it is defined in the conf?

Comment (calixteman) [reply auto_nag/scripts/leave_open_no_activity.py]: ok

**Pull request merged**